torch/distributed/c10d/distributed_c10d.py

```diff
@@ -373,14 +373,16 @@ def get_world_size(group=group.WORLD):
 
 def isend(tensor,
           dst,
-          group=group.WORLD):
+          group=group.WORLD,
+          tag=0):
     """
     Sends a tensor asynchronously.
 
     Arguments:
         tensor (Tensor): Tensor to send.
         dst (int): Destination rank.
         group (ProcessGroup, optional): The process group to work on
+        tag (int, optional): Tag to match send with remote recv
```

Comment:
Please some more description on how the tag should be used, this will go to doc automatically